chapters/ja/chapter7/1.mdx

```diff
@@ -0,0 +1,32 @@
+<FrameworkSwitchCourse {fw} />
+
+# イントロダクション
+
+[第3章](/course/chapter3)では、テキスト分類のためにモデルを微調整する方法を学びました。この章では、以下のような一般的な自然言語処理タスクに取り組みます。
```

Comment:
I think we'll need to put the language code here for the cross-link to work 

```suggestion
[第3章](/course/ja/chapter3)では、テキスト分類のためにモデルを微調整する方法を学びました。この章では、以下のような一般的な自然言語処理タスクに取り組みます。
```

Reply:
@lewtun 
Yay! That's my first PR approval!

Thanks for checking.
I will include the link fix in the next pull request.